builtin/mainmenu/tab_local.lua

```diff
@@ -33,10 +33,30 @@ if enable_gamebar then
 		return game
 	end
 
+	-- Apply menu changes from given game
+	function apply_game(game)
+		core.set_topleft_text(game.name)
+		core.settings:set("menu_last_game", game.id)
```

Comment:
Shouldn't these two be set by mm_game_theme?

Reply:
Hmm, not really